packages/devtools_app/lib/src/memory/memory_screen.dart

```diff
@@ -77,25 +78,42 @@ class MemoryScreen extends Screen {
 class MemoryBody extends StatefulWidget {
   const MemoryBody();
 
+  static const ValueKey dartHeapTabKey = ValueKey('Dart Heap');
```

Comment:
Do we really need keys for these? I don't see the keys used anywhere.

Reply:
I'll remove them.